File: ChatApp/Send Message/Steps - Ruby/StepDefinition-SendMessage-TextString.rb
Given /^a user \"([^\"]*)\"$/" do |alice|
    # do something
end

Given /^I am logged in as \"([^\"]*)\"$/" do |alice|
    # do something
end

When /^I post the Text String (.+)$/ do |string|
    # do something
end

Then /^I should see the Text String (.+)$/ do |string|
    # do something
end

And /^a user \"([^\"]*)\"$/" do |bob|
    # do something
end

And /^\"([^\"]*)\"" should see the Text String (.+)$/ do |string, bob|
    # do something
end

File: ChatApp/Send Message/Steps - JavaScript/StepDefinition-SendMessage-TextString.js
module.exports = function () {

  this.Given(/^a user \"([^\"]*)\"$/, function (alice, callback) {
    callback.pending();
  });

  this.Given(/^I am logged in as \"([^\"]*)\"$/, function (alice, callback) {
    callback.pending();
  });

  this.When(/^I post the Text String (.+)$/, function (string, callback) {
    callback.pending();
  });

  this.Then(/^I should see the Text String (.+)$/, function (string, callback) {
    callback.pending();
  });

  this.And(/^a user \"([^\"]*)\"$/, function (bob, callback) {
    callback.pending();
  });

  this.And(/^\"([^\"]*)\" should see the Text String (.+)$/, function (string, , callbackbob, callback) {
    callback.pending();
  });

};